{"name": "新增用户", "status": "passed", "attachments": [{"name": "log", "source": "671a004e-cf1a-4461-974c-d193898c3218-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "inData", "value": "'{\"password\":\"\",\"password_confirm\": \"88888888\",\"username\":\"001\"}'"}, {"name": "exp", "value": "'{'code':500,'msg':\"Key:'userAddRequest.password'Error:Field validation for 'password' failed on the'required' tag\"}'"}], "start": 1604655339804, "stop": 1604655339829, "uuid": "4534c0c5-08d5-4c8d-b51a-265e8fe6bffb", "historyId": "7eb54adce62304271e7e440620b1c341", "testCaseId": "f56f10c6b6679a14e1ccd1e8f399d334", "fullName": "test_userManage.TestUserManage#test_add_user", "labels": [{"name": "suite", "value": "test_userManage"}, {"name": "subSuite", "value": "TestUserManage"}, {"name": "host", "value": "dadaxPC008"}, {"name": "thread", "value": "19320-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_userManage"}]}